dragonfly/implementations/r2daq.py

```diff
@@ -0,0 +1,1442 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+"""
+
+from __future__ import division
+
+import adc5g
+from copy import deepcopy
+from corr.katcp_wrapper import FpgaClient
+from datetime import datetime
+import logging
+import netifaces as ni
+from numpy import (
```

Comment:
This is really a style/personal preference thing, but I don't like importing lots of elements `from` a library, especially something like `abs` which is replacing an existing core-python function. Instead it is nice to either use `numpy.abs` etc (so it is clear which version of abs or whatever you are using), or you can shorten it if you like with `import numpy as np` and `np.abs`.